Modules/CIPPCore/Public/New-CIPPCAPolicy.ps1

```diff
@@ -29,6 +29,25 @@ function New-CIPPCAPolicy {
             }
         }
     }
+    # Function to check if a string is a GUID
+    function Test-IsGuid($string) {
+        return [guid]::tryparse($string, [ref][guid]::Empty)
+    }
+    # Helper function to replace group display names with GUIDs
+    function Replace-GroupNameWithId {
```

Comment:
Would it make sense to pull this to its own script? Might be worth making usable elsewhere